runtime/gc_modron_startup/mmparseXXgc.cpp

```diff
@@ -101,11 +100,9 @@ gcParseXXgcArguments(J9JavaVM *vm, char *optArg)
 		}
 		if (try_scan(&scan_start, "stw")) {
 			MM_Scheduler::initializeForVirtualSTW(extensions);
-#if defined(J9VM_GC_STACCATO)
```

Comment:
Change like this required OMR code to be fixed first

Reply:
Also I believe J9VM_GC_STACCATO define check should be replaced to another (modern) flag

Reply:
ok, this definition is under J9VM_GC_REALTIME flag already. So it is ok to just remove J9VM_GC_STACCATO here.
OMR code can be fixed later